### DISCUSSION TITLE
图片显示发送成功但是群里面不显示

### DISCUSSION THREAD

**Discussion by yg9538** [mamoe/mirai#870] [Q&A] [answered]
是二次发送的图片
[0m2021-01-17 22:52:01 V/Net 54524527: Send: ImgStore.GroupPicUp[0m
[0m2021-01-17 22:52:01 V/Net 54524527: Recv: FileExists(fileId=2983706854, file
Info=ImgInfo(fileMd5=[99, 121, -35, -34, -3, -43, 47, 45, 57, -12, -103, 25, -93
, -10, -12, -56], fileType=1000, fileSize=109387, fileWidth=0, fileHeight=0))[0
m
[0m2021-01-17 22:52:02 V/Net 54524527: Send: MessageSvc.PbSendMsg[0m
[0m2021-01-17 22:52:02 V/Net 54524527: Recv: MessageSvcPbSendMsg.Response.SUCCE
SS[0m
[0m2021-01-17 22:52:05 V/Bot 54524527: Group(1014547123) <- [mirai:image:{6379D
DDE-FDD5-2F2D-39F4-9919A3F6F4C8}.jpg][0m
[0m2021-01-17 22:52:05 V/Net 54524527: Send: ImgStore.GroupPicUp[0m
[0m2021-01-17 22:52:05 V/Net 54524527: Recv: FileExists(fileId=2814229910, file
Info=ImgInfo(fileMd5=[16, -94, 10, 74, 108, 66, 14, 81, 17, 51, 87, 87, -101, -9
7, -73, 15], fileType=1000, fileSize=59985, fileWidth=0, fileHeight=0))[0m
[0m2021-01-17 22:52:06 V/Net 54524527: Send: MessageSvc.PbSendMsg[0m
[0m2021-01-17 22:52:06 V/Net 54524527: Recv: MessageSvcPbSendMsg.Response.SUCCE
SS[0m
[0m2021-01-17 22:52:09 V/Bot 54524527: Group(1014547123) <- [mirai:image:{10A20
A4A-6C42-0E51-1133-57579B9FB70F}.jpg][0m

**Comment by sandtechnology** [comment, marked as answer]
请检查手机是否显示 否则为QQ风控所致

**Reply by yg9538** [reply]
谢谢
